### ISSUE TITLE
Страница всех городов

### ISSUE THREAD

**Issue by nicothin** [nicothin/akinita#3]
Макет: https://www.figma.com/file/m7tbooJfqSTppw57xfapCe/%D0%9B%D0%B8%D1%87%D0%BD%D1%8B%D0%B9-%D0%BA%D0%B0%D0%B1%D0%B8%D0%BD%D1%82%D0%B5%2F-Payment-flow?node-id=1628%3A8933

Вёрстка: https://nicothin.pro/akinita/all-cities.html

Вопросы:

1. Мы правда хотим на мобилке белый фон, на настольном серый?
2. В мобилке есть кнопка Search, прибитая к нижней части страницы. Что по клику?
3. Если сущностей на странице много и я с широким экраном, как мне найти искомый город/регион?

@vladimir075 @greeceruit @nadezdafrolova

**Comment by nadezdafrolova** [comment]
1. На мобильной тоже делаем серый.
2. Оставляем сверху глобальный поиск (как сейчас, в текущей версии на страницах rent/ sale). Там правда после того как ввести что-то в поиск, срабатывает зум и страницу не прокрутить. Посмотрите, пожалуйста. 
3. Не поняла вопроса. Поиска по этой странице у нас нет. 

По текущей верстке:
1)  вот этот текст (984 homes for sale 340 apartments for rent 807 holiday rentals) должен быть Medim, и это активные ссылки которые ведут на поисковую выдачу.  
2)Нужно увеличить расстояние между горизонтальными блоками. до 64px. 
3) картинка по высоте 200px

**Comment by nicothin** [comment]
Отступы поиска на мое усмотрение же?

@nadezdafrolova Вопрос по появившимся рядом с мобильным представлением экранам: это дубликат главной навигации? 
![image](https://user-images.githubusercontent.com/3499680/166976568-22c60aba-7604-4721-a19c-7bc7ee47bdeb.png)

**Comment by nicothin** [comment]
- Сделал фон серым
- Добавил сверху поиск (при указании "как сейчас" прошу выдавать ссылку и/или скрин)
- Контентный текст блочков сделал ссылками, сменил плотность с 500 (макет) на 300 (пожелание)
- Привел верт. отступ на большой ширине к макетным 64
- Привет картинки на большой ширине точно к макетным размерам

**Comment by nadezdafrolova** [comment]
> * Сделал фон серым
> * Добавил сверху поиск (при указании "как сейчас" прошу выдавать ссылку и/или скрин)
> * Контентный текст блочков сделал ссылками, сменил плотность с 500 (макет) на 300 (пожелание)
> * Привел верт. отступ на большой ширине к макетным 64
> * Привел картинки на большой ширине точно к макетным размерам

ага, отлично

**Comment by nadezdafrolova** [comment]
> Отступы поиска на мое усмотрение же?
> 
> @nadezdafrolova Вопрос по появившимся рядом с мобильным представлением экранам: это дубликат главной навигации? Зачем эти фреймы? ![image](https://user-images.githubusercontent.com/3499680/166976568-22c60aba-7604-4721-a19c-7bc7ee47bdeb.png)

это мобильная версия ветки регионы- номосы- муниципалитеты

**Comment by nadezdafrolova** [comment]
да, все верно.

Но смотрите, у нас есть мега меню и вот эти два экрана для веб 1) https://www.figma.com/file/m7tbooJfqSTppw57xfapCe/%D0%9B%D0%B8%D1%87%D0%BD%D1%8B%D0%B9-%D0%BA%D0%B0%D0%B1%D0%B8%D0%BD%D1%82%D0%B5%2F-Payment-flow?node-id=1627%3A7522

2) https://www.figma.com/file/m7tbooJfqSTppw57xfapCe/%D0%9B%D0%B8%D1%87%D0%BD%D1%8B%D0%B9-%D0%BA%D0%B0%D0%B1%D0%B8%D0%BD%D1%82%D0%B5%2F-Payment-flow?node-id=1627%3A8145

И для них нужна мобильная версия. Можем короткий созвон сделать, если нужно

**Comment by nicothin** [comment]
> да, все верно.
> 
> Но смотрите, у нас есть мега меню и вот эти два экрана для веб 1) https://www.figma.com/file/m7tbooJfqSTppw57xfapCe/%D0%9B%D0%B8%D1%87%D0%BD%D1%8B%D0%B9-%D0%BA%D0%B0%D0%B1%D0%B8%D0%BD%D1%82%D0%B5%2F-Payment-flow?node-id=1627%3A7522
> 
> 2. https://www.figma.com/file/m7tbooJfqSTppw57xfapCe/%D0%9B%D0%B8%D1%87%D0%BD%D1%8B%D0%B9-%D0%BA%D0%B0%D0%B1%D0%B8%D0%BD%D1%82%D0%B5%2F-Payment-flow?node-id=1627%3A8145
> 
> И для них нужна мобильная версия. Можем короткий созвон сделать, если нужно

@nadezdafrolova Да, лучше созвониться, не уверен, что точно понимаю как это должно работать. Как насчет созвона днем, 12 мая в телеге? (https://t.me/ngromov)

**Comment by nicothin** [comment]
@vladimir075 @greeceruit @nadezdafrolova

Завершил страницу: https://nicothin.pro/akinita/regions.html

**Comment by vladimir075** [comment]
@greeceruit вот на этой странице мне кажется мы нагородили. поэтому сильно ей пока не заморачивайся, посмотрим как будем делать вместе при встрече
